tools/SetupFlow/DevHome.SetupFlow/ViewModels/QuickstartPlaygroundViewModel.cs

```diff
@@ -545,17 +590,36 @@ public void ProvideFeedback(bool isPositive, string feedback)
         _quickStartProject?.FeedbackHandler?.ProvideFeedback(isPositive, feedback);
     }
 
-    public bool CanGenerateCodespace()
-    {
-        return !string.IsNullOrEmpty(CustomPrompt) && ActiveQuickstartSelection != null;
-    }
-
     [RelayCommand]
     public void CopyExamplePrompt(string selectedPrompt)
     {
         CustomPrompt = selectedPrompt;
     }
 
+    public async Task GenerateChatStyleCompetions(string message)
+    {
+        if (ActiveQuickstartSelection != null)
+        {
+            _quickStartChatStyleGeneration = ActiveQuickstartSelection.CreateChatStyleGenerationOperation(message);
+            var result = await _quickStartChatStyleGeneration.GenerateChatStyleResponse();
+            ChatMessages.Add(new ChatStyleMessage
+            {
+                Name = result.ChatResponse.ToString(),
+                Type = ChatStyleMessage.ChatMessageItemType.Response,
+            });
+
+            if (result.ChatResponse.ToString().Contains("great prompt"))
+            {
+                IsPromptValid = true;

```

Comment:
what about a `great` prompt makes the prompt valid?  Additionally, I remember in the SDK you talking about what makes a valid prompt.

Who is responsible for determining if a prompt is valid? Extensions, or DevHome?